spyder/plugins/ipythonconsole/widgets/shell.py

```diff
@@ -1018,3 +1016,56 @@ def focusOutEvent(self, event):
         """Reimplement Qt method to send focus change notification"""
         self.sig_focus_changed.emit()
         return super(ShellWidget, self).focusOutEvent(event)
+
+    # --- value access --------------------------------------------------------
```

Comment:
I suggest that we move back the methods in this section to `NamepaceBrowserWidget`. They are only necessary for the Variable Explorer and not called by other methods here.

As with `DebuggingWidget` and the other widgets that compose `ShellWidget`, that allows us to have the size of `ShellWidget` manageable.

Reply:
Done